openpype/hosts/blender/plugins/load/load_image.py

```diff
@@ -31,17 +27,21 @@ def process_asset(self, context: dict, *args, **kwargs) -> bpy.types.Image:
         img = bpy.data.images.load(libpath)
         img.name = plugin.asset_name(asset, subset)
 
+        if libpath.endswith((".mov", "mp4")):
+            img.source = 'MOVIE'
+
         return img
 
 
 class ReferenceLoader(ImageLoader):
     """Load image as Reference."""
```

Comment:
Could you please explain a bit more the behaviour please? This way we can understand the difference between the loaders.

Reply:
doc updated